game-engine/Engine/Services/EngineService.cs

```diff
@@ -110,14 +119,17 @@ private async Task ProcessGameTick()
                 await hubConnection.InvokeAsync("PlayerConsumed", bot.Id);
             }
 
-            foreach (var bot in aliveBots)
-            {
-                Logger.LogDebug(bot.Id, "Size", bot.Size);
-                Logger.LogDebug(bot.Id, "Speed", bot.Speed);
-                Logger.LogDebug(bot.Id, "Position", $"{bot.Position.X}:{bot.Position.Y}");
-            }
+            stoplog.Log("Informed Consumed Bots");
```

Comment:
Maybe remove this?

Reply:
These are still needed from the cloud perspective for monitoring